### DISCUSSION TITLE
Per-app language selection for TeamTalk on Android

### DISCUSSION THREAD

**Discussion by Nardol** [BearWare/TeamTalk5#1623] [Ideas]
Hello,

Since Android 13, it is possible to select language per app in the phone settings->System->Languages.

Could it be possible to have it implemented in TeamTalk for Android?

There is documentation here: https://developer.android.com/guide/topics/resources/app-languages

**Comment by beqabeqa473** [comment]
will be fixed in #2035

**Comment by beqabeqa473** [comment]
Merged to master.
